### DISCUSSION TITLE
Trilhas Sonoras de Jogos… no Baixo

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#489] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/trilhas-sonoras-de-jogos-no-baixo/

**Comment by the-coffee-monk** [comment]
Não entendo nada de música, mas, seja lá qual for a função do baixo na composição de uma banda, o cara manda bem

**Comment by maryon-telvanni** [comment]
> [The Coffee Monk](</perfil/?user=the-coffee-monk>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/videos/trilhas-sonoras-de-jogos-no-baixo/#comment_1173886798>):

> Não entendo nada de música, mas, seja lá qual for a função do baixo na composição de uma banda, o cara manda bem

Também não entendo nada, mas, se é que dá pra botar fé nesses *snippets* do Google, ele faz a ponte entre os agudos (guitarra) e a percussão (bateria)
